region/brinstar/green/Spore Spawn Room.json

```diff
@@ -75,6 +75,11 @@
                       "Spazer"
                     ]}
                   ]},
+                  {"and": [
```

Comment:
This case (Ice+Wave+Spazer) seems redundant, as it's already covered by other cases above (e.g. Ice+Wave)?